src/main/java/com/google/sps/servlets/UserDataServlet.java

```diff
@@ -29,8 +31,20 @@
 public class UserDataServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType(MediaType.TEXT_PLAIN);
+    response.setContentType(MediaType.APPLICATION_JSON);
     response.getWriter().print(
-        UserServiceFactory.getUserService().getCurrentUser().getEmail());
+        createJSON());
+  }
+
+  /**
+   * Returns a JSON string containing the user's email.
+   */
+  private String createJSON() {
```

Comment:
This method should only have one function (separation of concerns). Right now it however does both 
 - retrieve user's email
 - serialize into JSON

Instead consider changing this to:
`private String createJson(String email);`

Reply:
Also rename the method to `createJson` or similar. See camelCase definition in the style guide: https://abc.xyz/investor/static/pdf/20200731_alphabet_10Q.pdf?cache=f16f989